src/formatDate.js

```diff
@@ -48,9 +48,36 @@
  *
  * @returns {string}
  */
-
+/* eslint-disable no-console */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
+  const FORMAT_DIVIDER_INDEX = 3;
+  const MAX_YEAR_TO_APPEAR = 30;
+  const TWENTIETH_CENTURY = '20';
+  const NINETEENTH_CENTURY = '19';
+  const oldDate = date.split(fromFormat[FORMAT_DIVIDER_INDEX]);
+  const oldDateCopmonets = {};
+  const dateNew = [];
+
+  for (let i = 0; i < FORMAT_DIVIDER_INDEX; i++) {
+    oldDateCopmonets[fromFormat[i]] = oldDate[i];
+  }
+
+  if (oldDateCopmonets.YY) {
+    if (oldDateCopmonets.YY < MAX_YEAR_TO_APPEAR) {
+      oldDateCopmonets.YYYY = TWENTIETH_CENTURY + oldDateCopmonets.YY;
+    } else {
+      oldDateCopmonets.YYYY = NINETEENTH_CENTURY + oldDateCopmonets.YY;
+    }
```

Comment:
Try to avoid nested conditions.
```suggestion
    const century = oldDateCopmonets.YY < MAX_YEAR_TO_APPEAR ? TWENTIETH_CENTURY : NINETEENTH_CENTURY;
    oldDateCopmonets.YYYY = century + oldDateCopmonets.YY;
```